src/framework/xr/xr-plane.js

```diff
@@ -159,10 +161,12 @@ class XrPlane extends EventHandler {
     }
 
     /**
-     * Array of DOMPointReadOnly objects. DOMPointReadOnly is an object with `x y z` properties
-     * that defines a local point of a plane's polygon.
+     * Gets the array of points that define the polygon of the plane in its local coordinate space.
```

Comment:
"an array". I believe the very first occurrence of a thing uses "a", and "the" afterwards.

Reply:
The points are not created in response to calling the getter. The array lives on the WebXR XRPlane and the getter simply returns them.